server/src/main/java/org/elasticsearch/rest/action/search/RestMultiSearchAction.java

```diff
@@ -137,7 +138,12 @@ public static MultiSearchRequest parseRequest(
             searchRequest.source(SearchSourceBuilder.fromXContent(parser, false));
             RestSearchAction.checkRestTotalHits(restRequest, searchRequest);
             if (searchRequest.pointInTimeBuilder() != null) {
-                RestSearchAction.preparePointInTime(searchRequest, restRequest, namedWriteableRegistry);
+                if (restRequest.paramAsBoolean("ccs_minimize_roundtrips", false)) {
+                    throw new IllegalArgumentException("[ccs_minimize_roundtrips] cannot be used with point in time");
+                }
+                if (searchRequest.indicesOptions().equals(DEFAULT_INDICES_OPTIONS) == false) {
+                    throw new IllegalArgumentException("[indicesOptions] cannot be used with point in time");
+                }
```

Comment:
I wonder why we need these at the REST layer, shouldn't these checks be performed as part of the request validation anyways. Would we fail anyways down the line without these additional checks?